scripts/kind-e2e/lib_helm_deploy_subm.sh

```diff
@@ -56,9 +56,12 @@ function setup_broker() {
 
 function helm_install_subm() {
     cluster_id=$1
-    cluster_cidr=$2
-    service_cidr=$3
-    crd_create=$4
+    crd_create=$2
+    cluster_cidr=$3
+    service_cidr=$4
+    global_cidr=$5
+    globalnet_enable="true" && [[ -z $global_cidr ]] && globalnet_enable="false"
```

Comment:
I think this will be more comprehensible:

```suggestion
    if [ -z $global_cidr ]; then globalnet_enable="false"; else globalnet_enable="true"; fi
```

Reply:
my previous commit used to have this. I updated it based on @vthapar's comments. I too agree that its better to use if ... then .. else ... which is more readable.
@vthapar hope you are ok :)

Reply:
I agree.  I found this line difficult to comprehend - we want it readable for non-scripting experts too 😄